stac_fastapi/mongo/setup.py

```diff
@@ -0,0 +1,57 @@
+"""stac_fastapi: mongo module."""
+
+from setuptools import find_namespace_packages, setup
+
+with open("README.md") as f:
+    desc = f.read()
+
+install_requires = [
+    "attrs",
+    "pydantic[dotenv]",
+    "stac_pydantic==2.0.*",
+    "stac-fastapi.types",
+    "stac-fastapi.api",
+    "stac-fastapi.extensions",
+    "shapely",
```

Comment:
is shapely really used? (I couldn't find after a quick 👀)

Reply:
It was used in a test - I switched to use geojson_pydantic like you suggested. Thank you.